libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/CriResponseItem.java

```diff
@@ -25,6 +26,7 @@ public class CriResponseItem implements DynamodbItem {
     private Instant dateCreated;
     private String status;
     private long ttl;
+    private List<String> featureSet;
```

Comment:
(I assume this works fine for existing records)

Reply:
Tested with already existing f2f records in dev.